lib/stupidedi/transaction_sets/004010/implementations/IN810.rb

```diff
@@ -0,0 +1,191 @@
+                                                                       b::Element(e::Situational, "Tax Type Code"),
+                                                                       b::Element(e::Situational, "Monetary Amount"),
+                                                                       b::Element(e::Situational, "Percent"),
+                                                                       b::Element(e::Situational, "Tax Jurisdiction Code Qualifier"),
+                                                                       b::Element(e::Situational, "Tax Jurisdiction Code"),
+                                                                       b::Element(e::Situational, "Tax Exempt Code"),
+                                                                       b::Element(e::Situational, "Relationship Code"),
+                                                                       b::Element(e::Situational, "Dollar Basis For Percent"),
+                                                                       b::Element(e::Situational, "Tax Identification Number"),
+                                                                       b::Element(e::Situational, "Assigned Identification")
+                                                            ),
+                                                            d::LoopDef.build("PID", d::RepeatCount.bounded(1000),
+                                                                             b::Segment(30, s::PID, "Product/Item Description", r::Situational, d::RepeatCount.bounded(1),
+                                                                                        b::Element(e::Situational, "Item Description Type"),
+                                                                                        b::Element(e::Situational, "Process Characteristic Code"),
+                                                                                        b::Element(e::Situational, "Agency Qualifier Code"),
+                                                                                        b::Element(e::Situational, "Product Description Code"),
+                                                                                        b::Element(e::Situational, "Description"),
+                                                                                        b::Element(e::Situational, "Surface/Layer/Position Code"),
+                                                                                        b::Element(e::Situational, "Source Subqualifier"),
+                                                                                        b::Element(e::Situational, "Yes/No Condition or Response Code"),
+                                                                                        b::Element(e::Situational, "Language Code")
+                                                                             )
```

Comment:
Consecutive closing parentheses should be written side-by-side (and should not be on their own line). Eg, `b::Element(..., ...)))`